presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeTpchConnectorQueries.java

```diff
@@ -41,16 +41,14 @@ protected void assertQuery(String sql)
         PrestoSparkNativeQueryRunnerUtils.assertShuffleMetadata();
     }
 
-    // TODO: Enable following Ignored tests after fixing (Tests can be enabled by removing the method)
     @Override
-    @Ignore
-    public void testMissingTpchConnector() {}
-
-    @Override
-    @Ignore
-    public void testTpchDateFilter() {}
-
-    @Override
-    @Ignore
-    public void testTpchTinyTables() {}
+    public void testMissingTpchConnector()
```

Comment:
Why can't we use the test from the base class?

Reply:
The error message is different for presto native and presto on spark native. I have moved the code around a bit to avoid the duplicate.

Reply:
Any particular reason for the difference? Should we make it the same?

Reply:
Typical spark deployment (including fb-spark) does not support querying multiple catalogue under the same spark session (unless it is a dataset app). So the expected error message is accurate for both native as well as pos native.

Reply:
I'd think this limitation would not apply to PoS since it uses PrestoSQL for language, query planning and execution and uses Spark only for distributed execution. Why would that piece care about whether there are multiple catalogs or just one?

Reply:
I see. This is quite unfortunate. Let's add a comment to explain this and let's also revisit this design decision.

Reply:
Does that means we're configuring the C++ worker with different connector name (tpch) from the connector name using on java side (hive)? 
https://github.com/prestodb/presto/blob/master/presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java#L376

Reply:
On java side we also configured with tpch connector name (client side we can have multiple connection names and most likely that is how it all worked) https://github.com/prestodb/presto/blob/master/presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java#L341